### DISCUSSION TITLE
建议添加电源功率传感器信息

### DISCUSSION THREAD

**Discussion by Maka8ka** [XiaoMi/ha_xiaomi_home#228] [Ideas]
同一个设备相较于xiaomi MIOT Auto项目缺少电源功率等传感器信息，建议添加
![Snipaste_2024-12-19_15-03-49](https://github.com/user-attachments/assets/65721bd1-b9cd-4666-9bc2-2bc331b5f29b)
![Snipaste_2024-12-19_15-04-44](https://github.com/user-attachments/assets/6eaa1687-8ebd-4d0a-90fb-e93ecb5b12fd)

**Comment by lzcapp** [comment]
`Xiaomi Miot Auto`确实是有，但是我调用接口也没看到y用电量的信息，不知道它是从哪里拿的还是自己算的
![image](https://github.com/user-attachments/assets/96a29358-6be5-4c01-b7c3-ee113dd708ce)
